Userland/Libraries/LibWeb/DOM/Element.cpp

```diff
@@ -190,6 +190,11 @@ WebIDL::ExceptionOr<void> Element::set_attribute(FlyString const& name, String c
     // 5. Change attribute to value.
     attribute->change_attribute(value);
 
+    //6. If size was changed, then update layout.
```

Comment:
Don't add a comment like this—the comments are quotes from the spec (a link to which, at the top of the method). If you need to add a comment it should be prefixed with something like FIXME or TODO or HACK or whatever.